app/assets/stylesheets/partials/_table.scss

```
@@ -0,0 +1,10 @@
+table.govuk-table {
+  @include govuk-media-query($until: tablet) {
+  display: block;
+}
+  overflow-x: scroll;
+}
+
+.govuk-template {
+    overflow-x:hidden;
```

Also, another tiny spacing thing:

```suggestion
  overflow-x:hidden;
```